src/pages/PasswordRecovery/index.tsx

```
@@ -10,93 +10,158 @@ import { Alert, AlertDescription } from "@/components/ui/alert"
+            email: "",
+        },
+    })
+
+    const onSubmit = async (data: PasswordRecoveryFormValues) => {
+        try {
+            const res = await forgotPassword(data.email);
+            console.log('Send email success:', res);
+            if (res.access_token) {
+                localStorage.setItem('token', res.access_token);
+            }
         } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)